client/my-sites/plans/current-plan/jetpack-checklist/index.js

```diff
@@ -148,17 +156,35 @@ class JetpackChecklist extends PureComponent {
 						);
 					} ) }
 				</Checklist>
+
+				{ wpAdminUrl && (
+					<JetpackChecklistFooter
+						wpAdminUrl={ wpAdminUrl }
+						handleWpAdminLink={ this.handleWpAdminLink }
+					/>
+				) }
 			</Fragment>
 		);
 	}
 }
 
 export default connect(
 	state => {
+		const site = getSelectedSite( state );
 		const siteId = getSelectedSiteId( state );
 		const productInstallStatus = getJetpackProductInstallStatus( state, siteId );
 		const rewindState = get( getRewindState( state, siteId ), 'state', 'uninitialized' );
 
+		// Link to "My Plan" page in Jetpack
+		let wpAdminUrl = get( site, 'options.admin_url', '' );
```

Comment:
Nit: For a falsey value that says "I don't have anything to put here", I'd prefer `null` or `undefined` over `""` (empty string).

Minor: Perf-wise, we're actually doing _a lot_ of work in `connect`, which can be a concern. This is probably fine, but we might consider reorganizing some of this to avoid doing so much work in `connect`.

_To be clear:_ All of this comment is non-blocking.

Reply:
Sure; I meant to keep it consistently as a string but `undefined` works for me 👍 

Changed in https://github.com/Automattic/wp-calypso/pull/32874/commits/909897384552e2bf34bb597a2ce0135c8addbc20

> Perf-wise, we're actually doing a lot of work in connect, which can be a concern. This is probably fine, but we might consider reorganizing some of this to avoid doing so much work in connect.

There might be point when expanding the checklist more when we'll want to optimize this indeed but that's for another PR.